package.json

```diff
@@ -16,7 +16,7 @@
     "jslint": "eslint lib tests scripts bin versionist.conf.js",
     "sasslint": "sass-lint lib/gui/scss",
     "htmllint": "node scripts/html-lint.js",
-    "codespell": "codespell.py --skip *.gz,*.bz2,*.xz,*.zip,*.img lib tests docs scripts Makefile *.md LICENSE",
+    "codespell": "codespell --dictionary=- --dictionary=dictionary --skip *.gz,*.bz2,*.xz,*.zip,*.img lib tests docs scripts Makefile *.md LICENSE",
```

Comment:
> an updated codespell package now installs with a different executable-name
(no .py extension), and now supports multiple dictionaries

Oh, I see. Just `codespell` doesn't seem to be working anymore though.